From: Dima Kogan <dima@secretsauce.net>
Date: Sat, 16 Nov 2013 20:56:24 -0800
Subject: printf format types now use 'z' when printing size_t data

---
 extra_matio.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/extra_matio.c b/extra_matio.c
index bd305c4..673cb0c 100644
--- a/extra_matio.c
+++ b/extra_matio.c
@@ -16,17 +16,17 @@ void extra_matio_print_all_var_info_clumsy(mat_t * matfp) {
     printf("%-20s", matvar->name);
     if ( matvar->rank > 0 ) {
       int cnt = 0;
-      printf("%8d", matvar->dims[0]);
+      printf("%8zd", matvar->dims[0]);
       for ( i = 1; i < matvar->rank; i++ ) {
         if ( ceil(log10(matvar->dims[i]))+1 < 32 )
-          cnt += sprintf(size+cnt,"x%d", matvar->dims[i]);
+          cnt += sprintf(size+cnt,"x%zd", matvar->dims[i]);
       }
       printf("%-10s",size);
     } else {
       printf("                    ");
     }
     nbytes = Mat_VarGetSize(matvar);
-    printf("  %8d",nbytes);
+    printf("  %8zd",nbytes);
     printf("         %-18s\n",mxclass[matvar->class_type-1]);
     Mat_VarPrint(matvar,0);
     Mat_VarFree(matvar);
